lib/camel_patrol/middleware.rb

```diff
@@ -18,7 +18,8 @@ def underscore_params(env)
       if ::Rails::VERSION::MAJOR < 5
         env["action_dispatch.request.request_parameters"].deep_transform_keys!(&:underscore)
       else
-        request_body = JSON.parse(env["rack.input"].read)
+        rack_input = env["rack.input"].read
+        request_body = JSON.parse(rack_input.length > 0 ? rack_input : '{}' )
```

Comment:
prob better to, if making any change, only translate valid json, otherwise let the invalid json though.

I'll update the pr to this --^